build.rs

```diff
@@ -8,7 +8,7 @@ use std::process::Command;
 use std::fs::File;
 use std::io::Write;
 
-use gcc::Config;
+use gcc::Build;
```

Comment:
We might need to this (and maybe other build.rs functions) behind a macos platform flag. I tested this branch with the following:

```
git remote add wez https://github.com/wez/nrf52dk-sys
git fetch wez
git checkout wez/master
rm -rf target
xargo build --release
```

I get this error on linux

![wez_master](https://user-images.githubusercontent.com/396004/32409421-b96e1a64-c168-11e7-9ef2-747bb97b628c.png)

Reply:
Hmmm, okay I take that back. I tried updating gcc and I no longer am missing `Build`, but I get other errors.

![upgraded_gcc](https://user-images.githubusercontent.com/396004/32409471-7e925292-c169-11e7-8e17-8f1382e49803.png)

Something funky in the generated code. Using these verions
`bindgen 0.31.3`
`xargo 0.3.8`